pkg/commands/resolver.go

```diff
@@ -78,8 +79,15 @@ func gobuildOptions(bo *options.BuildOptions) ([]build.Option, error) {
 		}
 	}
 
+	if bo.BaseImage != "" {
+		baseImageRef, err := name.ParseReference(bo.BaseImage)
+		if err != nil {
+			return nil, fmt.Errorf("'gobuildOptions': error parsing %q as image reference: %v", bo.BaseImage, err)
+		}
+		defaultBaseImage = baseImageRef
+	}
```

Comment:
```suggestion
```

Reply:
Fixed. Apologies, missed this in the rebase.